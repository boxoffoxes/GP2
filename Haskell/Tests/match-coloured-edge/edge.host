[
	(n1, empty)
	(n2, empty # red)
	(n3, empty # red)
	(n4, empty # green)
	(n5, empty # red)
	(n6, empty # red)
	(n7, empty # red)
|
	(e1, n1, n2, empty)
	(e2, n1, n3, empty)
	(e3, n1, n4, empty # dashed)
	(e4, n1, n5, empty)
	(e5, n1, n6, empty)
	(e5, n1, n7, empty)
]
