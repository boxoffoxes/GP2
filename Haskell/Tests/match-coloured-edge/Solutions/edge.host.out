[
 (n1, empty # red)
 (n2, empty # red)
 (n3, empty # red)
 (n4, empty # green)
 (n5, empty # red)
 (n6, empty # red)
 (n7, empty)
|
 (e1, n7, n1, empty)
 (e2, n7, n2, empty)
 (e3, n7, n3, empty)
 (e5, n7, n5, empty)
 (e6, n7, n6, empty)
]
